examples/DocQA/docker/chroma_start.sh

```diff
@@ -0,0 +1,3 @@
+pip install chromadb
```

Comment:
why is this needed? if you are starting from a chroma image you don't need it I think.

also can we inline this script into docker compose?

Reply:
Sure.. this has been changed